packages/performance/src/services/transport_service.ts

```diff
@@ -191,3 +129,13 @@ export function transportHandler(
     });
   };
 }
+
+/**
+ * Force flush the queued events. Useful at page unload time to ensure all
+ * events are uploaded.
+ */
+export function flushQueuedEvents(): void {
```

Comment:
Just to double check, regular traces/network requests should still be dispatched periodically before this flush is called right?

Reply:
That's right. The flush is only called after the oob trace is created. The other events still get dispatched at regular intervals.